### PR TITLE
Feature/events

### PR DESCRIPTION
**Pull request by fortuneonyeka**
In this milestone, I implemented the following:
- Copied the files calculate.js and operate.js into a logic/ directory.
- Analyze the files calculate.js and operate.js.
- Import the files into the Calculator component.
- Event handlers using the math logic from calculate.js and operate.js.

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    return {};
+  }
+
+  // Button must be an operation
+
+  // When the user presses an operation button without having entered
+  // a number first, do nothing.
+  // if (!obj.next && !obj.total) {
+  //   return {};
+  // }
```
Comment (DeliceLydia) [inline src/logic/calculate.js]: - [x] Please remove commented codes

File: src/logic/calculate.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+}
+
+/**
+ * Given a button name and a calculator data object, return an updated
+ * calculator data object.
+ *
+ * Calculator data object contains:
+ *   total:s      the running total
+ *   next:String       the next number to be operated on with the total
+ *   operation:String  +, -, etc.
```
Comment (DeliceLydia) [inline src/logic/calculate.js]: - [x] Same here you please remove commented codes

File: src/logic/calculate.js [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+ */
+export default function calculate(obj, buttonName) {
+  if (buttonName === 'AC') {
```
Comment (DeliceLydia) [inline src/logic/calculate.js:8]: - [x] I suggest that when you click this button you display a default zero in the screen and also when you open an application display a default zero

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @fortuneonyeka, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BrianSammit — APPROVED** [review]
# STATUS: APROVED :green_circle: :clap: 

Hello @fortuneonyeka, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**